fishy/gui/config_top.py

```diff
@@ -86,9 +94,22 @@ def toggle_sub():
     action_key_entry.grid(row=2, column=1)
     action_key_entry.insert(0, config.get("action_key", "e"))
 
-    Label(controls_frame, text="Sound Notification: ").grid(row=3, column=0, pady=(5, 5))
+
+    Label(controls_frame, text="Auto-Looting: ").grid(row=3, column=0, pady=(15, 0))
+    collect_allow_auto = Checkbutton(controls_frame, command=toggle_collect, var=BooleanVar(value=config.get("collect_allow_auto")))
+    collect_allow_auto.grid(row=3, column=1, pady=(15, 0))
+    
+    Label(controls_frame, text="Looting Key:").grid(row=4, column=0, pady=(0, 15))
+    collect_key_entry = Entry(controls_frame, justify=CENTER)
+    collect_key_entry.grid(row=4, column=1, pady=(0, 15))
+    collect_key_entry.insert(0, config.get("collect_key", "r"))
+    collect_key_entry['state'] = DISABLED
```

Comment:
use ternary operator instead to reduce code repetition and improve readability 

`collect_key_entry['state'] = NORMAL if config.get("collect_allow_auto") else DISABLED`

Reply:
Okay, I find this not very readable, but I don't mind either.